src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,92 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private final int initialCapacity = 16;
+    private final double loadFactor = 0.75;
+    private Node<K, V>[] table = new Node[initialCapacity];
+    private int size;
+    private int capacity = initialCapacity;
 
     @Override
     public void put(K key, V value) {
+        put(key, value,table);
+
+    }
 
+    private void put(K key, V value, Node<K, V>[] table) {
+        Node<K, V> node = new Node<>(key, value, null);
```

Comment:
if you never pass all three arguments, just create constructor for only `key` and `value`